src/App.js

```diff
@@ -7,36 +7,37 @@ import AddToDo from './components/AddToDo';
 import About from './components/About';
 import Header from './components/Header';
 import Login from './components/auth/Login';
-//import Auth from './components/auth';
+import Auth from './components/auth';
 import { Container, Navbar } from 'react-bootstrap';
 import { useState } from 'react';
 
-const data = [
+//const data = [
   //need to do a get here from the airtable
-  { id: 1, name: 'Autumn', title: 'Do the dishes, post and pans', assigned: 'Cat', difficulty: 1, completed: false },
-  { id: 2, name: 'Shaun T', title: 'Take the dog for a nice long walk.  Do 3 miles at least.', assigned: 'Scott', difficulty: 1, completed: false },
-  { id: 3, name: 'Joel', title: 'Run around the block and see what is up', assigned: 'Brianna', difficulty: 2, completed: false },
-  { id: 4, name: 'Chalene', title: 'Be beautiful', assigned: 'Brett', difficulty: 2, completed: false },
-  { id: 5, name: 'Tony', title: '200 sit ups', assigned: 'Tony', difficulty: 3, completed: false },
-];
+//  { id: 1, name: 'Autumn', title: 'Do the dishes, post and pans', assigned: 'Cat', difficulty: 1, completed: false },
+//  { id: 2, name: 'Shaun T', title: 'Take the dog for a nice long walk.  Do 3 miles at least.', assigned: 'Scott', difficulty: 1, completed: false },
+//  { id: 3, name: 'Joel', title: 'Run around the block and see what is up', assigned: 'Brianna', difficulty: 2, completed: false },
+//  { id: 4, name: 'Chalene', title: 'Be beautiful', assigned: 'Brett', difficulty: 2, completed: false },
+//  { id: 5, name: 'Tony', title: '200 sit ups', assigned: 'Tony', difficulty: 3, completed: false },
+//];
 let nextId = 50;
 
 
 
 function App() {
-  const [users, setUsers] = useState(data);
+  const [toDoItems, setToDoItems] = useState(ToDos);
```

Comment:
Found it. You're passing a component (which then calls other hooks) as the initial state. You should pass an empty array or null (Loading...)

So why does it show your odd error? Because if the initial value for `useState()` is a function then React calls that function when it need to set up the state the first time. Calling the component function then calls the hook functions, but only the first time (thus the error about a different number of hooks).